templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -413,7 +413,7 @@
 					<div class="df sb ac">
 						<div class="due-date tooltip {{if .Issue.IsOverdue}}text red{{end}}" {{if .Issue.IsOverdue}}data-content="{{.locale.Tr "repo.issues.due_date_overdue"}}"{{end}}>
 							{{svg "octicon-calendar" 16 "mr-3"}}
-							<time data-format="date" datetime="{{.Issue.DeadlineUnix.FormatLong}}">{{.Issue.DeadlineUnix.FormatDate}}</time>
```

Comment:
I am not quite sure why `FormatLong` and `FormatDate` should produce different results?

Reply:
Looks like `FormatDate` gives only the date, without HH:MM:SS (see example in PR description)

Reply:
Yeah, but both should still produce the correct date (`2023-01-04`), right?
So why should suddenly a day be added?
That's what I don't understand…

Reply:
`FormatLong` gives `Wed, 04 Jan 2023 23:59:59 +0000` so I think that when the date is localized it is either:
1. rounded
2. timezoned

It's just a guess though